tests/cephfs/cephfs_bugs/evict_all_clients_add_to_blocklist.py

```diff
@@ -0,0 +1,147 @@
+import random
+import string
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+"""
+ceph --admin-daemon $socketfile client evict [-h|--help] evicts ALL clients.
+It is observed that adding "--help|-h" with the "client evict" command
+terminates all active client connections and adds to the blocklist.
+ceph tell DAEMON_NAME client evict id=ID_NUMBER
+Steps to Reproduce:
+1. Connect clients to cephfs server
+2. # ceph --admin-daemon $socketfile client evict [-h|--help] evicts ALL clients
```

Comment:
We are not using this command in further automation code to evict client, but instead using  "ceph tell mds.cephfs.ceph-julpark-kg1mfg-node4.htpfxt client evict id=17763" as command. So, please help understand why have you mentioned "ceph --admin-daemon $socketfile client evict [-h|--help] evicts ALL clients" but using different command.

Reply:
[ceph: root@ceph-julpark-tb26v2-node5 /]# ceph --admin-daemon /var/run/ceph/ceph-mds.cephfs.ceph-julpark-tb26v2-node5.hqzhqj.asok
no valid command found; 10 closest matches:
0
1
2
abort
assert
cache drop [<timeout:int>]
cache status
client config <client_id:int> <option> [<value>]
client evict [<filters>...]
client ls [--cap-dump] [<filters>...]
admin_socket: invalid command
[ceph: root@ceph-julpark-tb26v2-node5 /]# ceph --admin-daemon /var/run/ceph/ceph-mds.cephfs.ceph-julpark-tb26v2-node5.hqzhqj.asok cache status
ERROR: (38) Function not implemented

[ceph: root@ceph-julpark-tb26v2-node5 /]# ceph --admin-daemon /var/run/ceph/ceph-mds.cephfs.ceph-julpark-tb26v2-node5.hqzhqj.asok client evict
ERROR: (38) Function not implemented

it lists all the possible cmds, but it says that the functions are not implemented